.circleci/config.yml

```diff
@@ -2770,12 +2770,12 @@ workflows:
             - discord
           requires:
             - initialize
-      # KURTOSIS (Isthmus)
+      # KURTOSIS (Jovian)
       - op-acceptance-tests:
           # Acceptance Testing params
-          name: kurtosis-isthmus
-          devnet: isthmus
-          gate: isthmus
+          name: kurtosis-jovian
+          devnet: jovian
+          gate: jovian
```

Comment:
We should probably move the 'isthmus' gate into 'base' which, similarly to "jovian" here, is supposed to be what's on mainnet and should run/work against all devnets.
(I can do this; assuming we agree)

Reply:
Yes please!

Reply:
Here you go: https://github.com/ethereum-optimism/optimism/pull/17514